src/components/modular-content/rich-text/rich-text.scss

```diff
@@ -0,0 +1,30 @@
+.rich-text {
+  a {
+    text-decoration: none;
+    color: clr(link);
```

Comment:
hoe werkt dit?

Reply:
ah dat is een functie van Zowie - sloop ik er wel even uit